src/thematic/vegetatie/svp-403_vegetatieopname-habitat-3260/svp-403_vegetatieopname-habitat-3260.Rmd

```diff
@@ -0,0 +1,189 @@
+---
+title: Vegetatieopname en LSVI-bepaling habitat 3260
+subtitle: 
+author: An Leyssen
+date: 2017-02-07
+output: 
+  bookdown::html_document2:
+    toc: true
+    toc_float: true
+    number_sections: true
+---
+
+<!--
+Instructies bij opmaak van veldprotocol:
+
+Hieronder vind je de verschillende onderdelen van een veldprotocol en een beschrijving wat er bij elk onderdeel vermeld moet worden. 
+Niet alle onderdelen zijn altijd van toepassing. Laat in die gevallen de titelhoofdingen staan en vermeld expliciet “niet van toepassing”.
+Probeer het veldprotocol generiek te maken in de mate van mogelijk en probeer te vermijden dat het veldprotocol enkel specifiek voor een bepaald project bruikbaar is. 
+Je kan het protocol generiek houden door bepaalde variabelen, zoals bijvoorbeeld de grootte van een proefvlak of de diepte van een bodembemonstering, niet vast te leggen in het protocol. 
+In bijlage kan je dan een tabel toevoegen waarin je per project aangeeft welke waardes gekozen moeten worden voor die variabelen. 
+Indien een bepaald onderdeel niet van toepassing is, laat je de hoofding staan en vermeld je hieronder ‘Niet van toepassing’. Op die manier is de nummering van alle protocols dezelfde.
+-->
+
+```{r setup, include=FALSE}
+library(knitr)
+opts_chunk$set(
+  echo = FALSE, 
+  eval = TRUE,
+  dpi = 300,
+  fig.width = 150 / 25.4,
+  fig.height = 100 / 25.4,
+  out.width = '100%',
+  warning = FALSE,
+  error = TRUE,
+  message = FALSE
+)
+```
+
+
+```{r metadata}
+# pas deze tabel aan
+metadata <- data.frame(
+  Reviewers = "Luc Denys, Toon Westra, Hans Van Calster",
+  Documentbeheerder = "Toon Westra",
+  Procedure = "svp-403",
+  Revisie = "1.0"
+)
+
+kable(metadata)
+
+```
+
+
+# Wijzigingen vorige versies
+
+<!--
+Omschrijf hier welke wijzigingen er gebeurd zijn ten opzichte van de vorige versie. Vermeld de datum van de vorige versie, de reden van de wijziging en een beknopte beschrijving van de wijziging.
+-->
+
+# Afhankelijkheden (Indien van toepassing)
+
+<!--
+Procedures waarnaar in deze procedure verwezen wordt: bv SIP_xxx
+Procedures die verwijzen naar deze procedure: bv SVP_xxx
+-->
```

Comment:
If dependency is set up more deeply (see general review) and encapsulates other (parts of) protocols, this part could be auto-generated.

Reply:
Yes, that would be ideal! Let's see if I can make this work.